refactor/urls.js

```diff
@@ -1,16 +1,26 @@
 // Refactor this code
 
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+  var url = "/tasks";
+  var params = new URLSearchParams();
+  if (dev) {
+    params.append("status", "ACTIVE");
+    params.append("dev", "true");
+    params.append("size", "20");
+  }
 
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
-    }
+  if (tasks.nextTasks) {
+    params.append("hasNext", "true");
+  }
 
-    if (tasks.prevTasks) {
-        url = '/tasks?hasPrev=true';
-    }
-    return { url };
+  if (tasks.prevTasks) {
+    params = new URLSearchParams();
```

Comment:
Is this logic correct?

Reply:
seeing the main code
```js
if (tasks.prevTasks) {
        url = '/tasks?hasPrev=true';
}
```
What I see is we should return the URL with "hasPrev" only and not include dev params. If the code in itself is wrong and we should also return the dev params, then we don't have to create new URLSearchParams() on line 17.